pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -118,21 +118,35 @@ func TestConfigPath(t *testing.T) {
 	const nondefaultPath = "/this/is/not/the/default/registries.conf"
 	const variableReference = "$HOME"
 	const rootPrefix = "/root/prefix"
+	oldHomeEnv, hasHomeEnv := os.LookupEnv("HOME")
+	err := os.Setenv("HOME", os.TempDir())
+	assert.Nil(t, err)
+	defer func() {
+		if hasHomeEnv {
+			os.Setenv("HOME", oldHomeEnv)
+		} else {
+			os.Unsetenv("HOME")
+		}
+	}()
+	var userRegistriesFile = filepath.FromSlash(".config/containers/registries.conf")
+	userRegistriesFilePath := filepath.Join(os.TempDir(), userRegistriesFile)
 
 	for _, c := range []struct {
 		sys      *types.SystemContext
 		expected string
+		userfile bool
```

Comment:
```suggestion
		userFilePresent bool
```

Reply:
… and please put it before the `expected` member, so that inputs go before outputs.